dev/Materials/Reveal/APITests/RevealTests.cs

```diff
@@ -40,8 +39,8 @@ public void ValidateAppBarButtonRevealStyles()
                         <CommandBar xmlns='http://schemas.microsoft.com/winfx/2006/xaml/presentation'
                             DefaultLabelPosition='Right'
                             IsOpen='True'>
-                            <AppBarButton Label='button' Icon='Accept' Style='{StaticResource AppBarButtonRevealLabelsOnRightStyle}'/>
```

Comment:
Are these styles no longer around? Why did you remove this?

Reply:
They're the default styles apparently. I could add them back for clarity if you think that's worth it, but the two should be functionally the same.

Reply:
@jevansaks Let me know if you think they should stick around and I'll do that before checking in, otherwise I'll go ahead. Thanks!

Reply:
I assumed if it was there before it was for a good reason, but if you're sure it's the default then it seems fine.